### DISCUSSION TITLE
请教一下AWS的CloudFront怎么设置

### DISCUSSION THREAD

**Discussion by chinanoahli** [v2fly/v2ray-core#568] [Q&A] [answered]
我采用了ws+tls的设置进行服务器搭建，最近域名过期了，所以打算换个域名
但是发现之前一直可用的CloudFront似乎不是太可以了，直连服务器的A记录域名可以正常使用
但是如果连接CloudFront的CDN CNAME之后会返回400 Bad Request

**Comment by imcaoxuan** [comment, marked as answer]
我也有这个问题

**Reply by chinanoahli** [reply]
> 我也有这个问题

![image](https://user-images.githubusercontent.com/10005128/108652781-4d2e4c00-7508-11eb-8e9f-80d885cca2f0.png)
![image](https://user-images.githubusercontent.com/10005128/108652795-53bcc380-7508-11eb-8a1a-e8f202c6f114.png)
使用旧版的CDN行为设置才可以，新版的无法设置（或者只是我不会设置而已）

**Reply by chinanoahli** [reply]
TTL根据自己需要填就可以
